### ISSUE TITLE
Auto-accept all changes (via command-line argument)

### ISSUE THREAD

**Issue by grp** [facebook/codemod#36]
It would be great to have a flag to auto-accept all changes

**Comment by modocache** [comment]
Good idea! :+1: 

There is such an option in interactive mode. It'd be great to have that be a command-line flag as well--something like `--autoaccept`?

**Comment by grp** [comment]
Is there? If so I missed it.

**Comment by modocache** [comment]
Yep, while running, you should see the following prompt:

```
Accept change (y = yes [default], n = no, e = edit, A = yes to all, E = yes+edit)?
```

`A` is what you're looking for--but you can only choose it once you're in interactive mode.

**Comment by grp** [comment]
Nice. Sounds like this is already there.

**Comment by modocache** [comment]
It would be great to be able to accept all changes *before* launching the script.

**Comment by modocache** [comment]
Fixed, thanks to #47. :checkered_flag:
